docs/manage/ml/_index.md

```diff
@@ -5,11 +5,25 @@ weight: 40
 type: "docs"
 tags: ["data management", "data", "services"]
 no_list: true
-description: "Use Viam's built-in machine learning capabilities to train image classification models and deploy these models to your robots"
+description: "Use Viam's built-in machine learning capabilities to train image classification models and deploy these models to your robots."
 # SME: Aaron Casas
 ---
 
-You can use Viam's built-in machine learning capabilities to [train image classification models](train-model/) and [deploy these models to your robots](../../services/ml/).
+Viam includes a built-in [machine learning (ML) service](/services/ml/) which provides your robot with the ability to learn from data and adjust its behavior based on insights gathered from that data.
+Common use cases include object detection, image classification, natural language processing, and speech recognition and synthesis, but your robot can make use of machine learning with nearly any kind of data.
+
+Viam natively supports [TensorFlow Lite](https://www.tensorflow.org/lite) ML models, but you can use other models as well as long as your models adhere to the [model requirements](/services/ml/#tflite_cpu-limitations).
+
+To make use of ML models with your robot, you can use the built-in ML model service to train image classification models for object detection and classification or create a [modular resource](/program/extend/modular-resources/) to integrate it with your robot.
+
+### Object detection and classification
+
+Once you have [created the ML model service](/services/ml/#create-an-ml-model-service) for your robot, you can [train image classification models](train-model/) to enable it to detect people, animals, plants or other objects with bounding boxes and perform actions when they are detected.
+
+![Gif of a dog being labeled](/tutorials/img/pet-treat-dispenser/app-data-images.png)
```

Comment:
Can also use the image Arielle has added as part of https://github.com/viamrobotics/docs/pull/1382/files.

Technically: both pictures are of the Data service showing a bunch of images that will shortly be used to train, not the actual training-of.

Reply:
sounds good. can use that until we can create something better

Reply:
could consider https://www.youtube.com/watch?v=CP14LR0Pq64